src/scss/atomic/molecules/_tooltip.scss

```diff
@@ -3,9 +3,8 @@
     background-color: $colorBlack;
     color: $colorWhite;
     font-weight: 400;
-    min-height: $buildingUnit * 3;
-    min-width: $buildingUnit * 8;
-    max-width: 150px;
+    min-width: $buildingUnit * 15;
+    max-width: 75%;
```

Comment:
The tooltip has a max-width that is defined in the design. Now it inherits something from the parent, which isn't reusable on the long term.